### ISSUE TITLE
Fall back to streaming_extractor_music in PATH if it exists

### ISSUE THREAD

**Issue by Kernald** [adamjakab/BeetsPluginXtractor#23]
### Use case

`streaming_extractor_music` already present in `$PATH` - this simplifies the configuration slightly

### Solution

If the configuration key isn't set and the binary is in `$PATH`, silently use it rather than failing. Otherwise, still fail as currently.

**Comment by JOJ0** [comment]
Please post an example on how the config would look like with your proposed feature.

**Comment by JOJ0** [comment]
@Kernald let me rephrase why it's not entirely clear to me what your feature request is: Are you talking about the stream extractor binary only ooor you mean that we would potentially find svm models as well in that place?

**Comment by Kernald** [comment]
I was just referring to the binary, as any folder in the `$PATH` should be a pretty standard location to store those binaries. As far as I know, there's no such standard location for the SVM models.

The reason I opened this issue is that I had the extractor binary in a folder in my `$PATH`, got an error about the binary being not found and was quite surprised, it's more about expected behaviour than simplifying the configuration file I guess.
